### PR TITLE
Slight layout fixes to the char import popup

### PR DESCRIPTION
**Pull request by Wolfsblvt**
- Need to pull my "wider" popup type from the other branch, because 1/1 aspect ratio scaling is just bad
- Add a few tailwind-like classes for ul-li styling
- ~~Add &lt;kbd&gt; element styling for keyboard buttons~~

image

### CODE REVIEW

File: public/scripts/templates/importCharacters.html [hunk 1]
```diff
@@ -19,7 +19,7 @@ <h4 data-i18n="Enter the URL of the content to import">Enter the URL of the cont
     <span data-i18n="Write each URL or ID into a new line.">
         Write each URL or ID into a new line.
     </span>
-    <span data-i18n="(Shift + Enter to start a new line)">
-        (Shift + Enter to start a new line)
+    <span data-i18n="(&lt;kbd&gt;Shift&lt;/kbd&gt; + &lt;kbd&gt;Enter&lt;/kbd&gt; to start a new line)">
```
Comment (Cohee1207) [inline public/scripts/templates/importCharacters.html]: Text inserted from i18n is escaped so this will not work

Comment (Wolfsblvt) [reply public/scripts/templates/importCharacters.html]: What do I have to do to "make this work"? Switch it into separate `<span>` blocks for the text too? :/

Comment (Cohee1207) [reply public/scripts/templates/importCharacters.html]: Remove this passage altogether? I don't think that power users would have problems with inserting a newline. Given that not even the main text input has this prompt.

Comment (Wolfsblvt) [reply public/scripts/templates/importCharacters.html]: Sure boss. 🫡

**Pull request merged**